docs/developer/issue-triage-process.md

```diff
@@ -0,0 +1,90 @@
+# Kubeapps triage process
+
+## Kubeapps backlog
+
+Kubeapps keeps a backlog of issues on GitHub submitted both by maintainers and contributors: this backlog comprises bugs, feature requests, and technical debt. If it’s a bug, an idea for a new feature, or something in between, it’s filed as an issue in the [kubeapps issue page](https://github.com/kubeapps/kubeapps/issues) on GitHub.
```

Comment:
Not sure about `’` vs `'`.

Reply:
https://guides.github.com/features/mastering-markdown/ See the examples for quotes there. So far we've always just used the normal `'` and let GH/Markdown smart quotes handle it (whether or not it does that correctly).